src/commands/schema/push.js

```diff
@@ -9,6 +9,20 @@ const FILE_LIMIT = 256;
 const FILESIZE_LIMIT_BYTES = 32 * 1024 * 1024;
 
 class PushSchemaCommand extends SchemaCommand {
+  async confirm() {
+    const resp = await ux.prompt("Accept and push the changes?", {
```

Comment:
```suggestion
    const resp = await ux.prompt("Accept and push the changes? (y/N)", {
```

Reply:
Because of the default, the prompt shows up as

Accept and push the changes? [no]:

so putting (y/N) in there is redundant.

Reply:
putting `[no]` in there is more confusing though, as its not clear what the options are

Reply:
If our users can't figure out this is a yes or no question, we are in big trouble.